docs/docs.mk

```diff
@@ -1,99 +1,51 @@
-
-.PHONY: docs-update-usage
-docs-update-usage: ## Generates dumps from `k8up --help`, which are then included as part of the docs
-	go run $(K8UP_MAIN_GO) --help > "$(docs_usage_dir)/k8up.txt"
-	go run $(K8UP_MAIN_GO) restic --help > "$(docs_usage_dir)/restic.txt"
-	go run $(K8UP_MAIN_GO) operator --help > "$(docs_usage_dir)/operator.txt"
-
-.PHONY: docs-generate-api
-docs-generate-api: $(crd_ref_docs_bin) ## Generates API reference documentation
-	$(crd_ref_docs_bin) --source-path=api/v1 --config=docs/api-gen-config.yaml --renderer=asciidoctor --templates-dir=docs/api-templates --output-path=$(CRD_DOCS_REF_PATH)
-
-.PHONY: docs-generate
-docs-generate: docs-update-usage docs-generate-api
-
-# This will clean the Antora Artifacts, not the npm artifacts
-.PHONY: docs-clean
-docs-clean: ## Cleans Antora artifacts
-	rm -rf $(out_dir) '?' .cache $(crd_ref_docs_bin)
+# Determine whether to use podman
+#
+# podman currently fails when executing in GitHub actions on Ubuntu LTS 20.04,
+# so we never use podman if GITHUB_ACTIONS==true.
+use_podman := $(shell command -v podman 2>&1 >/dev/null; p="$$?"; \
+		if [ "$${GITHUB_ACTIONS}" != "true" ]; then echo "$$p"; else echo 1; fi)
+
+ifeq ($(use_podman),0)
+	engine_cmd  ?= podman
+	engine_opts ?= --rm --userns=keep-id
+else
+	engine_cmd  ?= docker
+	engine_opts ?= --rm --user "$$(id -u)"
+endif
+
+use_go := $(shell command -v go 2>&1 >/dev/null; echo "$$?")
+
+ifeq ($(use_go),0)
+	go_cmd ?= go
+else
+	go_cmd ?= $(engine_cmd) run $(engine_opts) -e GOCACHE=/k8up/.cache -w /k8up --volume "$${PWD}:/k8up" golang:1.20 go
+endif
+
+orphans_cmd ?= $(engine_cmd) run $(engine_opts) --volume "$${PWD}:/antora" ghcr.io/vshn/antora-nav-orphans-checker:1.1 -antoraPath /antora/docs
+vale_cmd ?= $(engine_cmd) run $(engine_opts) --volume "$${PWD}"/docs/modules/ROOT/pages:/pages --workdir /pages ghcr.io/vshn/vale:2.15.5 --minAlertLevel=error .
+preview_cmd ?= $(engine_cmd) run --rm --publish 35729:35729 --publish 2020:2020 --volume "${PWD}":/preview/antora ghcr.io/vshn/antora-preview:3.1.2.3 --antora=docs --style=k8up
 
 .PHONY: docs-check
 docs-check: ## Runs vale against the docs to check style
+	$(orphans_cmd)
 	$(vale_cmd)
 
-.PHONY: docs-syntax
-docs-syntax: docs-html ## Runs hunspell against the docs
-	$(hunspell_cmd)
-
-.PHONY: docs-htmltest
-docs-htmltest: docs-html docs-pdf docs-epub docs-kindle docs-manpage ## Runs htmltest against the docs
-	$(htmltest_cmd)
-
 .PHONY: docs-preview
 docs-preview: ## Start documentation preview at http://localhost:2020 with Live Reload
 	$(preview_cmd)
 
-.PHONY: docs-html-open
-docs-html-open: docs-html ## Start documentation preview at http://localhost:2020 with Live Reload and open in browser
-	$(shell xdg-open _public/index.html)
-
-.PHONY: docs-html docs-pdf docs-manpage docs-epub docs-kindle
-docs-html:    $(out_dir)/index.html ## Generate HTML version of documentation with Antora, output at ./_public/
```

Comment:
There targets are not needed anymore?

Reply:
As explained in the summary. No, and they were broken for 2 years without bothering anyone